### PR TITLE
LPS-176251 Focus style and size is not homogeneous in control menu / product menu

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
[Link to ticket](https://issues.liferay.com/browse/LPS-176251)

## Test Scenario
* Create a new site.
* Go to the Product Menu.
* Check the icon to go to other site or library.

image

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by ethib137** [comment]
Hi @BarbaraCabrera, it looks like your fix is correctly handling the size of the site picker button, but it's not accounting for the difference in the focus state of some of the buttons. I've outlined in red the buttons that have an incorrect focus state in the below image. Let me know if you have any questions. 😄 
focus

**Comment by BarbaraCabrera** [comment]
> Hi @BarbaraCabrera, it looks like your fix is correctly handling the size of the site picker button, but it's not accounting for the difference in the focus state of some of the buttons. I've outlined in red the buttons that have an incorrect focus state in the below image. Let me know if you have any questions. 😄 focus

My bad! Thanks for the comment, I'll send a fix for the focus 😄

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/product-navigation/product-navigation-control-menu-theme-contributor/src/main/resources/META-INF/resources/control_menu/_control_menu.scss [hunk 1]
```diff
@@ -30,6 +30,7 @@ html#{$cadmin-selector} {
 		}
 
 		a.control-menu-icon,
+		button.control-menu-icon,
```
Comment (sandrodw3) [inline modules/apps/product-navigation/product-navigation-control-menu-theme-contributor/src/main/resources/META-INF/resources/control_menu/_control_menu.scss:33]: What do you think about merging this block of CSS with the previous one that also applies to `control-menu-icon`? 😄

**Review by sandrodw3 — COMMENTED** [review]
This is cool! Just left a tiny suggestion so we can take adventage and simplify the CSS a bit 🥳

**Comment by marcoscv-work** [comment]
This is cool, thanks guys.

What actually would be perfect is using buttons classes that already have focus style applied, example:

From this:
image


to this:
image


But we con go ahead with `.control-menu-icon` and in a future.. working on a control-menu refactor, add the button classes and remove the no longer needed styles in the control menu CSS

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
> This is cool, thanks guys.
> 
> What actually would be perfect is using buttons classes that already have focus style applied, example:
> 
> From this: image
> 
> to this: image
> 
> But we con go ahead with `.control-menu-icon` and in a future.. working on a control-menu refactor, add the button classes and remove the no longer needed styles in the control menu CSS

Hi, @marcoscv-work !
Thanks for the feedback, we opened a [technical debt](https://issues.liferay.com/browse/LPS-176799) for this 😄.

**Comment by BarbaraCabrera** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
ci:stop

**Comment by sandrodw3** [comment]
ci:stop please, I beg you 🙏

**Comment by BarbaraCabrera** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
ci:forward

**Pull request closed without merging**